spec/rubocop/cop/style/safe_navigation_spec.rb

```diff
@@ -950,6 +950,26 @@
           expect(new_source).to eq(expected_source)
         end
 
+        it 'only moves comments that fall within the expression' do
+          new_source = autocorrect_source(<<~RUBY)
+            # comment one
+            def foobar
+              if #{variable}
+                # comment 2
+                #{variable}.bar
+              end
+            end
+          RUBY
+
+          expect(new_source).to eq(<<~RUBY)
+            # comment one
+            def foobar
+              # comment 2
+            #{variable}&.bar
```

Comment:
This indentation seems a little weird because I think this is only place where it appears off. `Layout/IndentationWidth` will take care of this though.

Reply:
I think this would have happened before my change as well.